### PR TITLE
The PR that shows some love for admin tools

### PR DESCRIPTION
**Pull request by MaxIsJoe**
This PR adds a bunch of new QoL improvements for admin tools. Here are the changes : 

### Replacing pages with tabs

image

Pages were clunky and forced admins to click on multiple things to get to the page they needed, this has been fixed by switching to a "tab" approach where admins can click on the tab they want to switch to quickly at the top of the window.

### Improved readability and organization of some items

image
image
image

A lot of UI elements have been re-organized and were tweaked to be much more readable.

### Window transparency 

image

Admins can now set their window's transparency via a slider so they can keep an eye on the action while they still work from their tools. To show or hide this slider, press left shift and it will appear at the top like in the picture. 

### Other small non-visual QoL : 

- You will no longer have to worry about losing changes when closing or switching tabs
- Double click on a player entry to quickly admin ghost and orbit them.
- Fixed an issue where the quick respawn toggle would disappear, causing the respawn button to throw an NRE.
- When changing a player's job or giving them antag, you will go back to the player manager after you hit "confirm" 

CL: [Improvement] overhauled the admin tools menus to be easier to navigate.

### CODE REVIEW

File: UnityProject/Assets/Materials/Resources/OverlayCritHole.mat [hunk 1]
```diff
@@ -25,7 +25,7 @@ Material:
         m_Offset: {x: 0, y: 0}
     m_Ints: []
     m_Floats:
-    - _Radius: 5
```
Comment (corp-0) [inline UnityProject/Assets/Materials/Resources/OverlayCritHole.mat]: is this change intended?

File: UnityProject/Assets/Scripts/Managers/PlayerList.cs [hunk 2]
```diff
@@ -140,6 +140,17 @@ public List<ConnectedPlayer> GetPlayersOnMatrix(MatrixInfo matrix)
 		return InGamePlayers.FindAll(p => (p.Script != null) && p.Script.registerTile.Matrix.Id == matrix?.Id);
 	}
 
+	public ConnectedPlayer GetPlayerByID(string id)
+	{
+		foreach (var player in AllPlayers)
+		{
+			if(player.UserId != id) continue;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Managers/PlayerList.cs]: if you invert this, you won't need the ``continue`` making the algorithm have less exit points

File: UnityProject/Assets/Scripts/UI/Systems/AdminTools/GUI_AdminTools.cs [hunk 3]
```diff
@@ -60,28 +67,10 @@ public void ToggleOnOff()
 			gameObject.SetActive(!gameObject.activeInHierarchy);
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/UI/Systems/AdminTools/GUI_AdminTools.cs:67]: I'd assume a method called ``ClosePanel`` will always set the active state to false rather than invert the current state.

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/UI/Systems/AdminTools/GUI_AdminTools.cs:67]: you are confusing `ToggleOnOff()` with `ClosePanel()`

File: UnityProject/Assets/Materials/Resources/OverlayCritHole.mat (hunk 1 above)
Comment (MaxIsJoe) [reply UnityProject/Assets/Materials/Resources/OverlayCritHole.mat]: no, reverted.

**Comment by MaxIsJoe** [comment]
Approvals pls

**Review by Bod9001 — APPROVED** [review]
